### ISSUE TITLE
Extracting a multipart rar5 archive fails

### ISSUE THREAD

**Issue by Yentis** [files-community/Files#14328]
### Description

When attempting to extract a rar5 multivolume archive the extraction will fail, using regular 7zip works without a problem.
![7zFM_28ThIGEZwS](https://github.com/files-community/Files/assets/17981128/69d68d0a-fdc6-4000-bc00-321ebf40931e)

### Steps To Reproduce

1. Create a rar5 multivolume archive
2. Attempt to extract the first part using files

### Requirements

Fix extraction failing with multivolume archives

### Files Version

3.0.15.0

### Windows Version

21H2 19044.3693

### Log File

[debug.log](https://github.com/files-community/Files/files/13795841/debug.log)

**Comment by Josh65-2201** [comment]
Thanks for the report, I've added this to the project board.

**Comment by Josh65-2201** [comment]
@Yentis Is this still an issue on v4.0.11 https://files.community/download

**Comment by Yentis** [comment]
Yes, the status centre still shows a failure though no error is shown in debug.log

**Comment by Josh65-2201** [comment]
Thanks for checking

**Comment by yair100** [comment]
A fix will be included in the next release.
